src/bindings/xfconf/elektra-xfconf.h

```diff
@@ -0,0 +1,66 @@
+#ifndef ELEKTRA_XFCONF_H
+#define ELEKTRA_XFCONF_H
+
+#include <kdb.h>
+#include <xfconf/xfconf.h>
+
+#define I_(string) (g_intern_static_string ((string)))
+#define XFCONF_NAMESPACE "system:"
+#define XFCONF_ROOT "/sw/xfce4"
```

Comment:
Should be according guidelines in doc/tutorials/application-integration.md

Reply:
I will have a look on that. I hope that will not change that much.